packages/components/src/types.d.ts

```diff
@@ -10,6 +10,9 @@ export type { TextSize } from './components/text/text-size';
 export type { TextAlign } from './components/text/text-align';
 export type { TextColor } from './components/text/text-color';
 export type { TextWeight } from './components/text/text-weight';
+
+export type { HeaderAlign } from './components/carousel/carousel-align-header';
```

Comment:
This is only for shared/reused types which doesn't seem the case for this one.

Reply:
Fixed it